clang/test/SemaCXX/conditional-gnu-ext.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -std=c++11 %s
```

Comment:
I think this test should go in `clang/test/Sema/conditional-expr.c` and we should add a section of the GNU extension there.

I am also a bit concerned that we don't have a specific set of sema tests for the GNU form of the conditional expression. So it makes me much left confident that we will catch any breaks with this change.